apps/api/src/routes/uploads/handlers/create-upload.handler.ts

```diff
@@ -22,6 +22,17 @@ export async function createNewUploadHandler(
   }
 
   const keywords = file.buffer.toString().split("\n") || [];
+
+  if (keywords.map((keyword) => keyword.trim()).includes("")) {
+    res.status(400).json({
+      ok: false,
+      message:
+        "file must not contains any empty line or line that has only whitespace",
```

Comment:
An alternative approach would be not to error out but only process valid keywords. In terms of UX, spaces or empty lines could be considered human errors 💡

Reply:
That's also works. Let me update it to process only the valid keywords and disregard invalid lines

Reply:
Hi @olivierobert : I have updated the PR to filter out the empty, containing only whitespaces keywords and only process with the valid keywords. Please take a look.